code/datums/uplink/grenades.dm

```diff
@@ -5,27 +5,36 @@
 	category = /datum/uplink_category/grenades
 	
 /datum/uplink_item/item/grenades/manhack
-	name = "Manhack Delivery Grenade"
+	name = "Viscerator Delivery Grenade"
 	item_cost = 3
 	path = /obj/item/grenade/spawnergrenade/manhacks
+	desc = "A grenade that deploys five viscerator combat drones. Deadly in numbers, will not attack you or your allies."
 
 /datum/uplink_item/item/grenades/anti_photon
 	name = "5xPhoton Disruption Grenades"
 	item_cost = 4
 	path = /obj/item/storage/box/anti_photons
+	desc = "A box of five grenades that cause total darkness in the area they explode in. Circumvented by vision altering headgear. Useful for get-aways."
 
 /datum/uplink_item/item/grenades/smoke
 	name = "5xSmoke Grenades"
 	item_cost = 4
 	path = /obj/item/storage/box/smokes
+	desc = "A box of five grenades that deploy smoke in the thrown area. Targets hidden in smoke are much harder to hit with ranged weaponry."
 
 /datum/uplink_item/item/grenades/emp
 	name = "5xEMP Grenades"
 	item_cost = 6
 	path = /obj/item/storage/box/emps
+	desc = "A box of five grenades that cause a risky EMP explosion, capable of toggling headsets off, permanently destroying IPC units and draining a stationbound completely."
 	
 /datum/uplink_item/item/grenades/frag
 	name = "5xFrag Grenades"
 	item_cost = 12
 	path = /obj/item/storage/box/frags
 
+/datum/uplink_item/item/grenades/cardox
+	name = "5xCardox Grenades"
+	item_cost = 10
```

Comment:
Seems relative expensive compared to the EMP/Viscerator grenades.

Reply:
Cardox grenades are pretty toxic to all people except synths. I'll drop it down to 8.